app/grandchallenge/reader_studies/filters.py

```diff
@@ -1,7 +1,21 @@
+from django_filters import CharFilter
+from django_filters.rest_framework import FilterSet
+
 from grandchallenge.core.filters import TitleDescriptionModalityStructureFilter
-from grandchallenge.reader_studies.models import ReaderStudy
+from grandchallenge.reader_studies.models import Answer, ReaderStudy
 
 
 class ReaderStudyFilter(TitleDescriptionModalityStructureFilter):
     class Meta(TitleDescriptionModalityStructureFilter.Meta):
         model = ReaderStudy
+
+
+class AnswerFilter(FilterSet):
+    reader_study = CharFilter(
+        field_name="question__reader_study__title", label="reader study title"
```

Comment:
This breaks the existing API, we should only do that when we must. Please keep the key as `question__reader_study` and use the reader studies primary key as the value, I think that there should be a Related Model Filter rather than a `CharFilter`.

Reply:
There is actually a UUIDFilter that should do the trick, right? And so we want to keep the name of the field as "question__reader_study" as well, or is it fine to simplify to reader study?

Reply:
ah, I see now that we should keep the name at question__reader_study